docs/articles/user-guide/development-models.md

```diff
@@ -0,0 +1,209 @@
+   To change the default org you’re developing against, click the Org Picker and select a different org. Or, open the Command Palette and run **SFDX: Authorize an Org** or **SFDX: Create a Default Scratch Org**.
+
+### Push and Pull Source
+
+When you use the package development model, it’s simple to keep your local project and default development org in sync. Because you deploy your changes to other orgs using packaged sets of metadata, there’s no need to manually track your changes.
+VS Code is context aware that you are working in a scratch org and provides only push and pull commands, not commands to retrieve and deploy source.
+
+#### Push Source
+
+To push your source to the new scratch org or changes you made, run **SFDX: Push Source to Default Scratch Org**.
+
+If you want the changes in the project to overwrite changes in the scratch org, run **SFDX: Push Source to Default Scratch Org and Override Conflicts**.
+
+#### Pull Source
+
+After you make changes in your browser, run **SFDX: Pull Source from Default Scratch Org** to update your project.
+
+If you want the changes in the scratch org to overwrite changes in the project, run **SFDX: Pull Source from Default Scratch Org and Override Conflicts**.
+
+#### View Changes
+
+Before you push local changes to the scratch org or pull remote changes to the local project, you can see the changes in the Output panel. To do so, run **SFDX: View Changes in Default Scratch Org** from the Command Palette.
+
+## Salesforce CLI Commands
+
+Read on to learn about the CLI commands that Salesforce Extension for VS Code uses under the hood:
+
+- **SFDX: Create Project**
+  - runs `force:project:create`
+  - runs `force:project:create --template standard --manifest` if you are creating a project with manifest.
+- **SFDX: Authorize an Org**
+  - runs `force:auth:web:login --setdefaultdevhubusername` and sets the authenticated org as the default Dev Hub org for scratch org creation.
+  - runs `force:auth:device:login --setdefaultdevhubusername` if you are in remote development environment.
+- **SFDX: Create a Default Scratch Org** runs `force:org:create --setdefaultusername` and sets the created org as the default username.
+- **SFDX: Retrieve Source in Manifest from Org** runs `force:source:retrieve --manifest` where manifest parameter specifies file path of the components to retrieve.
+- **SFDX: Retrieve Source from Org** runs `force:source:retrieve`
+- **SFDX: Retrieve This Source from Org** runs `force:source:retrieve --sourcepath` and retrieves source from the org for the specified source file.
+- **SFDX: Deploy Source in Manifest in Org** runs `force:source:deploy --manifest` where manifest parameter specifies the complete file path of the components to deploy.
```

Comment:
What is the order of these commands? Shouldn't they be alphabetical? Maybe they should be grouped as well.

Reply:
Grouped the commands. Please review.